Sources/NetworkService/Request.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+/// Encapsulates an HTTP request with the necessary components.
+public struct Request {
+    public typealias HTTPHeaderKey = String
+    public typealias HTTPHeaderValue = String
+    public typealias HTTPHeaders = [HTTPHeaderKey: HTTPHeaderValue]
+
+    /// Creates a request with the provided parameters.
+    ///
+    /// - Parameters:
+    ///    - URI: The URL of the request.
+    ///    - requestMethod: The HTTP request method.
+    ///    - headers: HTTP header fields for the request.
+    ///    - body: The data sent as the message body of the request.
```

Comment:
Do we need to notice that it is `nil` by default?

Reply:
Fixed